scenarios/04_Valley_of_Trolls.cfg

```diff
@@ -308,12 +313,22 @@
         {RECALL_ADVISOR}
         [message]
             role=advisor
+            # TODO: spawn some additional Troll Whelps on the map in this event to make the "there are huge numbers of them" part truer:
             message= _ "We can’t get through, my Lord. These whelps are not individually very dangerous, but there are huge numbers of them."
         [/message]
 
         [message]
             speaker=Erlornas
+            # The turn limit varies with difficulty, so make this line of dialogue do likewise:
+#ifdef EASY
             message= _ "Four days of combat and nothing to show for it. How does it feel to you?"
+#endif
+#ifdef NORMAL
+            message= _ "Almost four days of combat and nothing to show for it. How does it feel to you?"
+#endif
+#ifdef HARD
+            message= _ "Just over three days of combat and nothing to show for it. How does it feel to you?"
```

Comment:
I think "Just over" sounds a bit too accurate, I suggest making it "Three days of combat and nothing to show for it ..."